gap/Internal.g

```diff
@@ -11,9 +11,31 @@ ListProduct@ := function(list)
     return p;
 end;
 
+# Helper function for SimplifyWord@,  keep track of multiple
+# function calling if the same groups are used in SimplifyWord@
```

Comment:
Do you mean multiple "function calls" or calls from multiple functions (which I wouldn't know how to interpret)?

```suggestion
# function calls if the same groups are used in SimplifyWord@
```